time-series/time-series-api/src/main/java/com/powsybl/timeseries/TimeSeries.java

```diff
@@ -213,8 +222,31 @@ private List<String> createStringValues() {
             return stringValues;
         }
 
-        int getVersion(String[] tokens) {
-            return timeSeriesCsvConfig.versioned() ? Integer.parseInt(tokens[1]) : 0;
+        int getVersion(String[] tokens, ReportNode reportNode) {
```

Comment:
we could allow the case versioned == true and version == DEFAULT_VERSION_NUMBER if DEFAULT_VERSION_NUMBER is the unique version of csv file

Reply:
Since `DEFAULT_VERSION_NUMBER_FOR_UNVERSIONED_TIMESERIES` is supposed to be for unversioned TimeSeries, I think it would not be a good idea to allow that case. If a user really wants to do it, s.he just has to set `strictVersioningImport` to false.
It's debatable but it's my point of view